memlab.bib

```diff
@@ -44067,7 +44147,7 @@ @article{MannEtal11
 	Title = {Oscillatory patterns in temporal lobe reveal context reinstatement during memory search},
 	Volume = {108},
 	Year = {2011},
-	Bdsk-File-1 = {YnBsaXN0MDDSAQIDBFxyZWxhdGl2ZVBhdGhZYWxpYXNEYXRhXxBJLi4vLi4vLi4vVm9sdW1lcy9Ucm9nZG9yMS9Vc2Vycy9wY3J1dGNobC9Ecm9wYm94L3BkZl9yZXBvL01hbm5FdGFsMTFhLnBkZk8RAZYAAAAAAZYAAgAACFRyb2dkb3IxAAAAAAAAAAAAAAAAAAAAAAAAAMbfsdVIKwAABJWenA9NYW5uRXRhbDExYS5wZGYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEla/my1bGfAAAAAAAAAAA/////wAACQAAAAAAAAAAAAAAAAAAAAAIcGRmX3JlcG8AEAAIAADG3+oVAAAAEQAIAADLVwzMAAAAAQAQBJWenAGPSHYABauKAACRngACADxUcm9nZG9yMTpVc2VyczoAcGNydXRjaGw6AERyb3Bib3g6AHBkZl9yZXBvOgBNYW5uRXRhbDExYS5wZGYADgAgAA8ATQBhAG4AbgBFAHQAYQBsADEAMQBhAC4AcABkAGYADwASAAgAVAByAG8AZwBkAG8AcgAxABIAMC9Vc2Vycy9wY3J1dGNobC9Ecm9wYm94L3BkZl9yZXBvL01hbm5FdGFsMTFhLnBkZgATABEvVm9sdW1lcy9Ucm9nZG9yMQD//wAAAAgADQAaACQAcAAAAAAAAAIBAAAAAAAAAAUAAAAAAAAAAAAAAAAAAAIK}}
+	Bdsk-File-1 = {YnBsaXN0MDDUAQIDBAUGJCVYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3ASAAGGoKgHCBMUFRYaIVUkbnVsbNMJCgsMDxJXTlMua2V5c1pOUy5vYmplY3RzViRjbGFzc6INDoACgAOiEBGABIAFgAdccmVsYXRpdmVQYXRoWWFsaWFzRGF0YV8QSS4uLy4uLy4uL1ZvbHVtZXMvVHJvZ2RvcjEvVXNlcnMvcGNydXRjaGwvRHJvcGJveC9wZGZfcmVwby9NYW5uRXRhbDExYS5wZGbSFwsYGVdOUy5kYXRhTxEBlgAAAAABlgACAAAIVHJvZ2RvcjEAAAAAAAAAAAAAAAAAAAAAAAAAxt+x1UgrAAAElZ6cD01hbm5FdGFsMTFhLnBkZgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAASVr+bLVsZ8AAAAAAAAAAD/////AAAJAAAAAAAAAAAAAAAAAAAAAAhwZGZfcmVwbwAQAAgAAMbf6hUAAAARAAgAAMtXDMwAAAABABAElZ6cAY9IdgAFq4oAAJGeAAIAPFRyb2dkb3IxOlVzZXJzOgBwY3J1dGNobDoARHJvcGJveDoAcGRmX3JlcG86AE1hbm5FdGFsMTFhLnBkZgAOACAADwBNAGEAbgBuAEUAdABhAGwAMQAxAGEALgBwAGQAZgAPABIACABUAHIAbwBnAGQAbwByADEAEgAwL1VzZXJzL3BjcnV0Y2hsL0Ryb3Bib3gvcGRmX3JlcG8vTWFubkV0YWwxMWEucGRmABMAES9Wb2x1bWVzL1Ryb2dkb3IxAP//AACABtIbHB0eWiRjbGFzc25hbWVYJGNsYXNzZXNdTlNNdXRhYmxlRGF0YaMdHyBWTlNEYXRhWE5TT2JqZWN00hscIiNcTlNEaWN0aW9uYXJ5oiIgXxAPTlNLZXllZEFyY2hpdmVy0SYnVHJvb3SAAQAIABEAGgAjAC0AMgA3AEAARgBNAFUAYABnAGoAbABuAHEAcwB1AHcAhACOANoA3wDnAoECgwKIApMCnAKqAq4CtQK+AsMC0ALTAuUC6ALtAAAAAAAAAgEAAAAAAAAAKAAAAAAAAAAAAAAAAAAAAu8=}}
```

Comment:
remove field

Reply:
This was a PDF of MannEtal11. Removed in latest commit.